src/api/TasksNew.js

```diff
@@ -13,6 +13,12 @@ import {
     API_PAGE_LIMIT,
 } from '../constants';
 
+// microservices need different headers than other APIs
+const headers = {
```

Comment:
Should these be a shared constant or is the duplication intentional?

Reply:
This should be a temporary workaround, these APIs should soon accept the standard json headers

Reply:
But I am not sure whether they will all be fixed at the same time. If this isn't resolved soon we can move to the constants file.